CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.7.2] - 2020-12-11
```

Comment:
Wrong version number in changelog :), currently 1.7.2 should be 1.7.3

Reply:
Copy and paste without adjusting :-) now it's fixed